README.rst

```diff
@@ -130,13 +130,13 @@ Bob must gather at least ``threshold`` cfrags in order to activate the capsule.
   # Several Ursulas perform re-encryption, and Bob collects the resulting `cfrags`.
   # He must gather at least `threshold` `cfrags` in order to activate the capsule.
 
-  capsule.set_correctness_keys(delegating=alices_public_key,
-                               receiving=bobs_public_key,
-                               verifying=alices_verifying_key)
+  prepared_capsule = capsule.with_correctness_keys(delegating=alices_public_key,
```

Comment:
This reads like a layer of indirection to me.  We had a "capsule", but if we add something to it, we get not the same "capsule", but a different "capsule"?  Of a different class?

Reply:
See above about having a `VerifyingMetadata` (or something along these lines) instead that would wrap these three keys. What do you think about this approach?

Reply:
And then compose that on this capsule?

I'm interested to hear about the *why* in addition to the *what*.  Why is this change (and the rest of the capsule fissure) helpful in your mind?

Reply:
Because the verifying metadata is secondary to the capsule. A capsule can be used without it. You can keep it as a separate object, or as a part of another object combining both the capsule and the keys, that's details. The change I'm introducing here is to remove the *mutability*. I do not understand why you are calling it a fissure.